pac4j-http/src/main/java/org/pac4j/http/credentials/TokenCredentials.java

```diff
@@ -42,4 +42,11 @@ public String getToken() {
     public String toString() {
         return CommonHelper.toString(this.getClass(), "token", this.token, "clientName", getClientName());
     }
+
+    @Override
+    public void clear() {
+        this.token = null;
+        this.setClientName(null);
+        this.setUserProfile(null);
```

Comment:
In fact, my suggestion might not be a good idea. The user profile saved into this credentials is passed this way between an `Authenticator` and a `ProfileCreator` to eventually return it.
So clearing the credentials would clear the user profile, which we may not want.